File: baekjoon/심화1/4344.cpp
#include<iostream>
using namespace std;

int main(){
    int c, n;
    double score[1000];
    cin >> c;
    for(int i=0; i<c; i++){
        cin >> n;
        int sum=0, count=0;
        for(int j=0; j<n; j++){
            cin >> score[j];
            sum+=score[j];
        }
        double avg = sum / n;
        for(int j=0; j<n; j++) if(score[j]>avg) count++;
        printf("%.3f%%\n", (double)count/n*100);
    }
}

File: baekjoon/심화1/4344.py
c = int(input())

for i in range(c):

    score = list(map(int, input().split()))
    avg = sum(score[1:]) / score[0]
    count=0

    for i in score[1:]:

        if avg<i: count+=1

    rate = count / score[0] * 100
    print("%.3f%%" %(rate))